docs/docs/admin/monitoring/logging/explorer.md

```diff
@@ -0,0 +1,119 @@
+and when you select the second label, you will only see labels that can be
+combined with the first filter. For example, if the first filter limits the
+logs to backend logs, you won't be able to select anything else, while if the
+first filter limits the logs to session logs, you can further restrict the
+tool, version, etc.
+
+The CCM supports a number of different labels and corresponding values, which
+are described below:
+
+-   **Deployments**: This is the label that should always be selected first
+    because it specifies the specific application from which you want to view
+    the logs. For example, you can select **-backend**, which would show all
+    backend logs when running, or **-sessions**, which would show all session
+    logs.
+-   **Session specific labels**: At the moment, we just attach more descriptive
+    labels to the session logs. Here we can further specify the user, session
+    type, tool, and/or version for which we want to see logs.
+
+Here is an example filter where we select all sessions for version 7.0.0.
+
+![Example Label Filter](./label_filter_example.png){:style="width:800px"}
+
+## Operations
+
+Second, we can define many different operations to apply to the logs selected
+by the label filters. Due to the large number of supported operations, we will
+only show examples that are often useful when using the Explorer.
+
+#### Line Contains
+
+The line contains operations that essentially search the log messages for the
+text you want to find.
+
+![Log Operations](./operations.png){:style="width:300px"}
+
+#### Backend Logfmt
+
+For our backend logs, we follow the [logfmt](https://brandur.org/logfmt), which
+provides more structure to the log lines and is especially useful when used
+with Loki/Grafana. The reason for this is that we can use the `Logfmt`
+operation to convert the key-value pairs in our logs to labels. These labels
+can then be used to filter the logs more fine-grained and efficiently.
+
+Here is an example operation sequence where we can query all logs that have the
+log level **INFO** and the HTTP method **POST**.
+
+![Operations Example](./operations_example.png){:style="width:500px"}
+
+Theoretically, we could achieve the same result by changing two line contains
+operations, but first, this is really inefficient since a string search has to
+be applied to all logs, and second, we have to use the exact format as in the
+log message.
+
+![Operations Example without Logfmt](./operations_example_bad.png){:style="width:400px"}
+
+## Time Range Selector
+
+Even though Loki is quite efficient, it is still not feasible to always query
+all the data, so you need to select the time range for which you want to see
+the logs, or use the default "Last 1 hour" time range. To change the time
+range, open the drop-down menu in the top right corner and select the time
+range you want to see.
+
+![Time Range Picker](./time_range.png){:style="width:300px"}
```

Comment:
This image looks very small to me, might be different on other screen setups but I would consider making it wider

Reply:
Changed it to 450px